server/settings/publish_plugins.py

```diff
@@ -720,7 +720,7 @@ class IntegrateHeroVersionModel(BaseSettingsModel):
 
 class CleanUpModel(BaseSettingsModel):
     _isGroup = True
-    paterns: list[str] = SettingsField(
+    patterns: list[str] = SettingsField(
```

Comment:
This was kept as `paterns` on purpose for backwards compatibility. This would require to handle the change of the model in server addon.